Article/Article.js

```diff
@@ -112,3 +144,67 @@ const data = [
   Step 5: Add a new article to the array. Make sure it is in the same format as the others. Refresh the page to see the new artible
 
 */
+
+const articles = document.querySelector(".articles");
+
+data.forEach(data => {
+  console.log(data.title);
+  articles.appendChild(
+    create(
+      data.title,
+      data.date,
+      data.firstParagraph,
+      data.secondParagraph,
+      data.thirdParagraph
+    )
+  );
+});
+
+////create elements
+function create(title, date, first, second, third) {
+  const articles = document.createElement("div");
+  const article = document.createElement("div");
+  const articleTitle = document.createElement("h2");
+  const articleDate = document.createElement("p");
+  const firstPara = document.createElement("p");
+  const secondPara = document.createElement("p");
+  const thirdPara = document.createElement("p");
+  const expandButton = document.createElement("span");
+
+  console.log(articleDate);
+
+  ////setup structure of elements
+  articles.appendChild(article);
+  article.appendChild(articleTitle);
+  article.appendChild(articleDate);
+  article.appendChild(firstPara);
+  article.appendChild(secondPara);
+  article.appendChild(thirdPara);
```

Comment:
I learned from Dora that you can give `appendChild` multiple arguments, so you could do `article.appendChild(firstPara, secondPara, thirdPara)`

Reply:
It turns out you should do that with `append` not `appendChild`